### DISCUSSION TITLE
unable to display the image from the details pane of the transaction code : /OPT/VIM_VA2

### DISCUSSION THREAD

**Discussion by VRK2281** [microsoft/openjdk#575] [Q&A] [answered]
hi, 

We are experiencing an issue in which we are unable to display the image from the details pane of the transaction code : /OPT/VIM_VA2 and a blank screen appears when we click on the Image button from the details pane.

There was no problem until yesterday. Last weekend we performed a migration from Oracle JRE (JAVA_VERSION="1.8.0_311") to Microsoft open JDK version: jdk-11.0.20.101-hotspot

Everything is working fine except for the above-mentioned error, after the migration.

**Comment by karianna** [comment]
See https://adoptium.net/en-GB/docs/migration/ for some common gotchas (including graphical issues) when moving off Oracle Java 8 to an OpenJDK alternative.

Is this an app you built yourself or a 3rd party?

**Reply by KalleOlaviNiemitalo** [reply, marked as answer]
A web search for /OPT/VIM_VA2 found [3064973 - Invoice image is unavailable in VIM Workplace and VIM Analytics by OpenText](https://userapps.support.sap.com/sap/support/knowledge/en/3064973) in SAP Knowledge Base.  The description there seems similar.
